src/Components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Button ({onClick, disabled, children})  {
+    return (
+        <>
+            <button
+                type="button"
+                onClick={onClick}
+                disabled={disabled||false}
+            >
+                {children}
+            </button>
+        </>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken